app/src/common/shared/com/igalia/wolvic/browser/SettingsStore.java

```diff
@@ -289,6 +289,21 @@ public void setTrackingProtectionLevel(int level) {
         mSettingsViewModel.setIsTrackingProtectionEnabled(level != WContentBlocking.EtpLevel.NONE);
     }
 
+    public static boolean shouldStartWithPassthrougEnabled() {
+        return DeviceType.getType() == DeviceType.LenovoA3;
```

Comment:
Could you please add Lynx to the list of devices that should have the switch on by default ?

Reply:
Done

Reply:
Haven't used Lynx-R1, but looks like it's a mixed-reality headset with a colored passthrough, so I think making the passthrough on by default can be reasonable? Anyway, this is an open question and we can revert to https://github.com/Igalia/wolvic/commit/28762826eb51fa61965ea24edec61d8ab20dcaf4 at anytime.

@javifernandez @svillar

Reply:
> The Meta Quest Pro does also have good color passthrough and same for the Lenovo VRX. That's not a reason to enable it by default. The reason why it's enabled by default in the A3 is because it's a see through device with a very limited FOV in front of the user. This means that the environment is almost useles because you can only barely see it behind the browser window. The rest of your FOV is the real world. So let's stick to the previous setting and have it only by default on the A3.

Okay, reasonable, reverted.

Reply:
I know I asked to remove Lynx but apparently the preferred option is to start with passthrough enabled as @javifernandez requested, so please bring it back. Sorry for the back and forth

Reply:
> I know I asked to remove Lynx but apparently the preferred option is to start with passthrough enabled as @javifernandez requested, so please bring it back. Sorry for the back and forth

@svillar 

But FYI, @javifernandez requested to remove Lynx again afterwards (in group chat):

```
There is a bug in lynx that causes all the android menu / dialogs to not be visible when passthrough is enabled. Should we revert (again, sorry) the change that adds lynx to the list of devices starting with passthrough enabled ? It's not caused but that, and it's a bug that needs to be fixed anyway, so that's why I'm asking 
```

So let's keep it as it is, we can add it back when that bug is addressed in another PR. (Sorry I can't help with that bug since I don't have a Lynx device, not sure what's going on there)

Reply:
Fair enough